index.js

```diff
@@ -1,109 +1,51 @@
-"use strict";
-
-const fs = require("fs");
+const fs = require("fs/promises");
```

Comment:
@Saiv46 Can you check why it fails on node 10.x? I suppose `require('fs').promises` could fix it though.

Reply:
My bad, named exports (like `fs/promises`) are supported since v12